provider/cloudflare/cloudflare.go

```diff
@@ -1021,6 +1100,16 @@ func groupByNameAndTypeWithCustomHostnames(records DNSRecordsMap, chs CustomHost
 			e = e.WithProviderSpecific(source.CloudflareCustomHostnameKey, strings.Join(customHostnames, ","))
 		}
 
+		if records[0].Comment != "" {
```

Comment:
not covered with unit tests

![Screenshot 2025-05-13 at 07 32 09](https://github.com/user-attachments/assets/30b3b5ec-3489-4931-a02f-adf4990d8d0c)

Reply:
covered